dojo/settings/settings.dist.py

```diff
@@ -226,6 +226,7 @@
     DD_MAX_REQRESP_FROM_API=(int, -1),
     DD_MAX_AUTOCOMPLETE_WORDS=(int, 20000),
     DD_JIRA_SSL_VERIFY=(bool, True),
+    DD_JIRA_DESCRIPTION_MAX_LENGTH=(int, 32768),
```

Comment:
Same here as above. Is this also elsewhere in the code? Hmm....

```suggestion
    DD_JIRA_DESCRIPTION_MAX_LENGTH=(int, 32767),
```